common.cmake

```diff
@@ -38,6 +38,13 @@ function(nrfxlib_calculate_lib_path lib_path)
   if(${CALC_LIB_PATH_SOC_MODE})
     # CMake regex does not support {4}
     string(REGEX REPLACE "_[a-zA-Z][a-zA-Z][a-zA-Z][a-zA-Z]$" "" arch_soc_dir ${CONFIG_SOC})
+
+    # Apply CPU cluster manually for devices that have multiple cores
+    if(DEFINED CONFIG_SOC_NRF5340_CPUAPP)
+      set(arch_soc_dir ${arch_soc_dir}_cpuapp)
+    elseif(DEFINED CONFIG_SOC_NRF5340_CPUNET)
```

Comment:
I believe this change will break all the libs that were renamed to `nrf5340/<abi>` in scope of https://github.com/nrfconnect/sdk-nrfxlib/pull/1279

Reply:
> they have been updated here

I missed that, thanks. The change looks good then